api/specs/web-server/_resource_usage.py

```diff
@@ -43,15 +45,29 @@
     tags=["usage"],
 )
 async def list_resource_usage_services(
-    wallet_id: WalletID = Query(None),
+    order_by: Annotated[
+        Json | None,
+        Query(
+            description="Order by field (started_at|stopped_at|credit_cost) and direction (asc|desc). The default sorting order is ascending.",
+            example='{"field": "started_at", "direction": "desc"}',
+        ),
+    ] = None,
+    filters: Annotated[
+        Json | None,
+        Query(
+            description="Filters to process on the resource usages list, encoded as JSON. Currently supports the filtering of 'started_at' field with 'from' and 'until' parameters in <yyyy-mm-dd> UTC format. The date range specidied is inclusive.",
```

Comment:
UTC stands for times, not for dates. I think what you mean is ISO 8601 format [https://en.wikipedia.org/wiki/ISO_8601](https://en.wikipedia.org/wiki/ISO_8601)
unless you can also add times? but anyway please adjust the documentation.

Reply:
Oh, I didn't realize that you're right! Thanks